### PR TITLE
Zabbix container docker-entrypoint.sh changed in 6.2.7

### LINKED ISSUE #29: database doesn't get initialized with version 6.2.7 when more than 1 replicas
The job `init-db-schema` which is supposed to start a zabbix-server-pgsql container and modify the docker-entrypoint.sh in a way that it just and only initializes a new database schema on an empty database does not work anymore. It exits after 0 seconds of runtime and does not output any logs. Database never gets initialized.

### PR DESCRIPTION
**Pull request by sa-ChristianAnton**
In 6.2.7, Zabbix has changed the structure of the docker-entrypoint.sh, causing the database initialization mechanism employed within this helm chart not to work anymore, when ramping up a Zabbix "cluster" with more than 1 replica.

Additionally, I have changed the retention policy of the db access secret not to be deleted on helm uninstall.

This PR fixes #29 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: charts/zabbix/templates/job-init-db-schema.yaml [hunk 1]
```diff
@@ -35,6 +35,6 @@ spec:
         command:
           - "/bin/sh"
           - "-c"
-          - 'sed -e "s/^exec \"\$@\"$/prepare_server/" -e "/^ *update_zbx_config$/d" /usr/bin/docker-entrypoint.sh >~/docker-entrypoint.sh && bash ~/docker-entrypoint.sh'
+          - 'export V=$(zabbix_server -V | head -n 1 | sed "s/.*\([0-9]\+\.[0-9]\+\.[0-9]\+\)/\1/"); if [ "$V" = "`echo -e "$V\n6.2.6" | sort -V | head -n1`" ]; then sed -e "s/^exec \"\$@\"$/prepare_server/" -e "/^ *update_zbx_config$/d" /usr/bin/docker-entrypoint.sh >~/docker-entrypoint.sh && bash ~/docker-entrypoint.sh; else /usr/bin/docker-entrypoint.sh init_db_only; fi'
```
Comment (BGmot) [inline charts/zabbix/templates/job-init-db-schema.yaml:38]: Won't this break all 6.2.0-5 versions?

Comment (BGmot) [reply charts/zabbix/templates/job-init-db-schema.yaml:38]: 6.0 is not affected?

Comment (sa-ChristianAnton) [reply charts/zabbix/templates/job-init-db-schema.yaml:38]: I don't get it. Why should it break these?

```
alias vercheck='if [ "$V" = "`echo -e "$V\n6.2.6" | sort -V | head -n1`" ]; then echo old; else echo new; fi'

export V=6.2.0; vercheck
old

export V=6.2.5; vercheck
old

export V=4.4.13; vercheck
old

export V=6.2.6; vercheck
old

export V=6.2.7; vercheck
new

export V=6.2.8; vercheck
new

export V=6.4.0; vercheck
new
```

I suppose that every version newer than 6.2.7 will also have this new "feature". Once we don't have to support any old (LTS) versions anymore (6.0...) this entire job manifest can be changed to not supply any command but only an argument `init_db_only`. But that's gonna take some while.

Comment (BGmot) [reply charts/zabbix/templates/job-init-db-schema.yaml:38]: Sorry, misinterpreted your bash, everything is great.

Comment (sa-ChristianAnton) [reply charts/zabbix/templates/job-init-db-schema.yaml:38]: yeah, it's a bit crazy, but AFAIK the most elegant way to compare versions in bash...

Comment (BGmot) [reply charts/zabbix/templates/job-init-db-schema.yaml:38]: I did not even know about `-V` in `sort`, very smart! -)

Comment (sa-ChristianAnton) [reply charts/zabbix/templates/job-init-db-schema.yaml:38]: Actually, I did find it on Stack overflow myself 😃

File: charts/zabbix/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+data:
+  dbname: emFiYml4
+  host: emJ4LXphYmJpeC1wb3N0Z3Jlc3Fs
+  password: cVZha1dydEdHSktKT0ZQRwo=
+  port: NTQzMg==
+  user: emFiYml4
```
Comment (aeciopires) [inline charts/zabbix/secret.yaml:7]: Hi @sa-ChristianAnton!
Can this values be dynamics like .values.PARAMETER_NAME ?

**Pull request merged**